examples/balance/app.ddlog

```diff
@@ -0,0 +1,12 @@
+yes (vote_id bigint).
+no  (vote_id bigint).
+
+whatever? (
+    candidate bigint
+).
+
+@weight(1)
+whatever(p) :- yes(_).
```

Comment:
Right. So they also need the question marks in the schema and can be given labels to act as evidence.